R/methods-mzRpwiz.R

```diff
@@ -192,10 +191,11 @@ setMethod("chromatogramHeader", "mzRpwiz",
               } else {
                   stopifnot(is.numeric(chrom))
                   n <- nChrom(object)
-                  chrom <- as.integer(chrom)
-                  if (min(chrom) < 1 || max(chrom) > n)
-                      stop("Index out of bound [", 1, ":", n, "]")
-                  chrom <- chrom - 1L
+                  cat("n=",n,"min(chrom)=",min(chrom)," max(chrom)=",max(chrom), "\n")
```

Comment:
can you please remove the debugging `cat()` calls here ;)

Reply:
Done.